### DISCUSSION TITLE
LC 4CH relay in shuttermode only showing one shutter and addressing wrong relays

### DISCUSSION THREAD

**Discussion by maartenjd** [arendst/Tasmota#11582] [Q&A] [answered]
Hello, I have an LC Technology X4 relay with an ESP8266-module (ESP-01S) and equipped with a Nuvoton N76E003AT20 host controller on the relay board.

I am using Tasmota 8.5.1, as from version 9 the entire GPIO-layout has changed.

I followed [these instructions](https://templates.blakadder.com/LC-ESP01-4R-12V.html) to configure the module. These apply (I suppose) for version 8 firmware's GPIO-layout. I tried Tasmota 9.3.1 with [another suggested template](https://tasmota.github.io/docs/devices/LC-Technology-WiFi-Relay/#lc-technology-wifi-relay-x2-with-nuvoton-n76e003at20) but this gives the same results as described below.

I enter the template in Configure Other as follows:
`{"NAME":"LC Technology 4CH Relay","GPIO":[21,0,22,0,23,24,0,0,0,0,0,0,0],"FLAG":0,"BASE":18}`

Further on I followed the instructions on the Tasmota-site [regarding shutters and blinds](https://tasmota.github.io/docs/Blinds-and-Shutters/):
```
SetOption80 1
ShutterMode 2
ShutterRelay1 1
ShutterRelay2 1
```

`ShutterPosition1` and `ShutterPosition2` commands in the console give the expected output (`RESULT = {"Shutterx":{"Position":0,"Direction":0,"Target":0}}`).

The GUI however shows like [this](https://www.dropbox.com/s/ckap1n9aa19mggy/Capture%20d%E2%80%99%C3%A9cran%202021-04-04%20%C3%A0%2014.47.59.png?dl=0). As you can see, it shows 2 open/close sliders, but only one pair of up/down buttons for the first shutter and  toggle buttons for the second pair of relays (3 and 4). Activating the shutter buttons works as it should (and activates relays 1 and 2 according ShutterMode 2 (circuit safe). The toggle-buttons toggle relays 3 and 4 respectively.

The console shows that slider 1 issues ShutterPosition1 commands and that slider 2 issues ShutterPosition2 commands, but they both switch relays 1 and 2.

Issuing `ShutterPosition1` and `ShutterPosition2` commands in the console [reveals](https://pastebin.com/RuxzE6pR) that only Power1 and Power2 are triggered whereas for Shutter2, Power3 and Power4 should be used.

Am I missing a step in the configuration or is this a bug in Tasmota?

**Comment by TrDA-hab** [comment, marked as answer]
It should be like this:

`SetOption80 1` // Enable shutter support
`ShutterMode 2` // Set "Shutter mode 2"
`ShutterRelay1 1` // for relay Relay1 and Relay2
`ShutterRelay2 3` // for relay Relay3 and Relay4
`Restart 1` // Restart Tasmota

Error in this command!
`ShutterRelay2 3` // for relay Relay3 and Relay4
![AAA](https://user-images.githubusercontent.com/56403720/113513072-7367f280-9570-11eb-9803-6cc300632287.jpg)

**Reply by maartenjd** [reply]
Thanks [TrDA-hab](https://github.com/TrDA-hab). It works. With this tip I now know how to read the ShutterRelay<x>-command explanation in the [commands guide](https://tasmota.github.io/docs/Commands/).
